### PR TITLE
Change package name for module mixed-format/spark

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Change package name to `org.apache.amoro` for mixed-format/spark modules.

## Brief change log

- Change package name.
- Change related configuration.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Review by Aireed — APPROVED** [review]

**Review by zhongqishang — COMMENTED** [review]
@zhoujinsong `.tokens` and `.interp` does not exist before, it is necessary to add its?

**Comment by zhoujinsong** [comment]
> @zhoujinsong `.tokens` and `.interp` does not exist before, it is necessary to add its?

Thanks for the detailed review. They should not be added. I will drop them.

File: mixed-format/spark/v3.3/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala [hunk 1]
```diff
@@ -16,7 +16,9 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.spark
+package org.apache.amoro.spark
+
+import org.apache.amoro.spark.{SparkAdapter, SparkAdapterLoader}
```
Comment (zhongqishang) [inline mixed-format/spark/v3.3/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala:21]: Unused import statement.

File: mixed-format/spark/v3.2/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala [hunk 2]
```diff
@@ -16,7 +16,9 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.spark
+package org.apache.amoro.spark
+
+import org.apache.amoro.spark.{SparkAdapter, SparkAdapterLoader}
```
Comment (zhongqishang) [inline mixed-format/spark/v3.2/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala:21]: Unused import statement.

File: mixed-format/spark/v3.3/spark/src/test/resources/log4j2.properties [hunk 3]
```diff
@@ -17,11 +17,11 @@
...
 
 log4j.rootLogger=INFO
-log4j.logger.com.netease.arctic=INFO
-log4j.logger.com.netease.arctic.spark=DEBUG
+log4j.logger.org.apache.amoro=INFO
+log4j.logger.org.apache.amoro.spark=DEBUG
 log4j.logger.org.apache.thrift.server=WARN
 log4j.logger.org.sparkproject.jetty.server=WARN
-log4j.logger.com.netease.arctic.ams.api.client=WARN
+log4j.logger.org.apache.amoro.client=WARN
```
Comment (zhongqishang) [inline mixed-format/spark/v3.3/spark/src/test/resources/log4j2.properties:24]:
```suggestion
log4j.logger.com.netease.arctic.client=WARN
```

File: mixed-format/spark/v3.2/spark/src/test/resources/log4j2.properties [hunk 4]
```diff
@@ -17,11 +17,11 @@
...
 
 log4j.rootLogger=INFO
-log4j.logger.com.netease.arctic=INFO
-log4j.logger.com.netease.arctic.spark=DEBUG
+log4j.logger.org.apache.amoro=INFO
+log4j.logger.org.apache.amoro.spark=DEBUG
 log4j.logger.org.apache.thrift.server=WARN
 log4j.logger.org.sparkproject.jetty.server=WARN
-log4j.logger.com.netease.arctic.ams.api.client=WARN
+log4j.logger.org.apache.amoro.client=WARN
```
Comment (zhongqishang) [inline mixed-format/spark/v3.2/spark/src/test/resources/log4j2.properties:24]:
```suggestion
log4j.logger.com.netease.arctic.client=WARN
```

**Comment by zhoujinsong** [comment]
Can you help to review again? @zhongqishang

**Review by zhongqishang — COMMENTED** [review]
I left some small comments, PTAL.

File: mixed-format/spark/v3.3/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala (hunk 1 above)
Comment (zhoujinsong) [reply mixed-format/spark/v3.3/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala]: Removed.

File: mixed-format/spark/v3.2/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala (hunk 2 above)
Comment (zhoujinsong) [reply mixed-format/spark/v3.2/spark/src/main/scala/org/apache/amoro/spark/SupportSparkAdapter.scala]: Removed.

**Review by zhongqishang — APPROVED** [review]
LGTM.

**Pull request merged**